src/chrome/content/rules/Moverall.com.xml

```diff
@@ -0,0 +1,15 @@
+<ruleset name="Moverall">
+
+	<target host="moverall.com" />
+	<target host="*.moverall.com" />
+
+	<test url="http://moverall.com/" />
+	<test url="http://www.moverall.com/" />
+	<test url="http://cdn.moverall.com/" />
+	<test url="http://az698912.vo.msecnd.net/" />
```

Comment:
No, it's done on a url-by-url basis. For each URL, first we do a fast match against target hosts, and then for all rulesets that match that way we check rule regexps.